forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -5061,16 +5067,23 @@ private boolean switchPhaseState(final boolean fromUntapStep) {
             // when it doesn't exist the game will no longer see it as tapped
             runUntapCommands();
             // TODO CR 702.26f need to run LeavesPlay + changeController commands but only when worded "for as long as"
+
+            // these links also break
+            clearEncodedCards();
+            if (isPaired()) {
```

Comment:
Doesn't paired only breaks if they don't phase out/in at the same time?

Reply:
Both Soulbond triggers have "for as long as both remain creatures on the battlefield under your control"

So it should expire imo

> 702.26f In particular, effects with “for as long as” durations that track that permanent (see rule 611.2b) end when that permanent phases out because they can no longer see it.